test/examples/test_kissgp_additive_classification.py

```diff
@@ -32,7 +32,7 @@ def __init__(self):
         self.mean_module = ConstantMean(prior=SmoothedBoxPrior(-1e-5, 1e-5))
         self.covar_module = ScaleKernel(
             RBFKernel(
-                ard_num_dims=2, log_lengthscale_prior=SmoothedBoxPrior(exp(-5), exp(6), sigma=0.1, log_transform=True)
+                ard_num_dims=1, log_lengthscale_prior=SmoothedBoxPrior(exp(-5), exp(6), sigma=0.1, log_transform=True)
```

Comment:
Looks like raising the error loudly is paying off already :-)

Maybe we should be explicit about how we intend for people to use ARD with additive structure in documentation

Reply:
@jacobrgardner this actually only affects variational additive structure, which we're going to deprecate (or significantly change) anyways.

For additive regression ard_num_dims works in the expected way (=2 for 2D additive)